.claude/hooks/python_check/file_extractor.py

```
@@ -1,14 +1,13 @@
-"""
+"""File path extraction from tool inputs and responses."""
 
-from typing import Any, Dict, List, Set, Tuple
-from .config import FILE_MODIFICATION_TOOLS, PYTHON_EXTENSIONS, MARKDOWN_EXTENSIONS
+from typing import Any
 
+from .config import FILE_MODIFICATION_TOOLS, MARKDOWN_EXTENSIONS, PYTHON_EXTENSIONS
 
-def _extract_paths_from_payload(payload: Any) -> List[str]:
+
+def _extract_paths_from_payload(payload: Any) -> list[str]:
```

Ruff (0.12.2)

Dynamically typed expressions (typing.Any) are disallowed in `payload`

(ANN401)